tests/Fixtures/Integration/misc/braces,phpdoc_indent.test

```diff
@@ -0,0 +1,14 @@
+--TEST--
+Integration of fixers: braces,phpdoc_indent.
```

Comment:
it fails, but not crash, when run over one fixer.
any fixer that would run after phpdoc_indent could crash, as phpdoc_intent leaves collection in invalid state. as told in other pr and here, this has nothing to do with that particular rules combination.